docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+
+By default, a class has `ref` capability. Therefore, these two class definitions are equivalent:
+
+```pony
+class ref RefClass
+    ...
+
+class RefClass
+    ...
+```
+
+But whether specified by default or explicitly as `class ref` (or `class val` or any other capability) what does it actually mean? A class or actor's default capability only applies when you _define_ a field or local type. This can happen either as `let` or `var`, or when specifying a parameter on a function or behaviour.
+
+## Field, Variable, and Parameter capabilities
+
+For example, `String` is defined in the standard library as `class val String`. So it's default capability is the immutable shareable `val` capability. That means that all the `String`s in the following actor definition could be replaced with `String val`:
+
+```pony
+actor UsesStrings
+    let s1: String
+    var s2: String
+
+    new create(s1': String, s2': String) =>
+        s1 = s1'
+        s2 = s2'
+
+    be do_thing(s3: String) =>
+        None
+
+```
+
+However `StringBytes` is defined as `class ref`, so all the `StringBytes`' in the following class definition are equivalent to `StringBytes ref`:
+
+```pony
+class UsesStringBytes
+    let s1: StringBytes
+    var s2: StringBytes
+
+    new create(s1': StringBytes, s2': StringBytes) =>
+        s1 = s1'
+        s2 = s2'
+
+    fun do_thing(s3: StringBytes) =>
+        None
+```
+
+Notably, however, the default capability on a class **does not** influence the constructors for that class.
+
+## Constructor capabilities
+
+Let's start with the easy defaults first: A primitive constructor _always_ returns an object with `val` capabilities, and an actor constructor _always_ returns an object with `tag` capabilities. In fact, it is forbidden to put a capability after `new` when defining a primitive or actor.
+
+Classes are a little more flexible. The capability returned by a `new` constructor default to a `ref` if you are constructing a `class`. This is true regardless of what the default capability for the class is, but you have the option to specify a different capability by explicitly typing the constructor as e.g `new var create` or `new iso create_iso`.
+
+Consider the following class:
+
+```pony
+class val MyClass
+    let foo: String
+
+    new create_ref(foo': String) =>
+        foo = foo'
+
+    new val create_val(foo': String) =>
+        foo = foo'
+```
+
+This class has two constructors. The first one, `create_ref` returns a `MyClass` with `ref` capabilities because `ref` is always default for class constructors. The second one `create_val` explicitly returns a `MyClass` with `val` capabilities.
+
+_Tip_: Instead of using the default `ref` type for constructors, consider using `iso`. Recall that constructors, like `consume` return their values as an ephemeral type. That means that a constructor typed as `new iso` actually returns a `iso^`. And `iso^` has the terrific property of being convertible to any other type! Consider:
```

Comment:
```suggestion
_Tip_: Instead of using the default `ref` capability for constructors, consider using `iso`. Recall that constructors, like `consume` return their values as an ephemeral type. That means that a constructor typed as `new iso` actually returns a `iso^`. And `iso^` has the terrific property of being convertible to any other property! Consider:
```

Reply:
I think the correct term here would be "to any other capability"

Reply:
Changed to `capability`, but I will leave this comment unresolved. Mainly, I need confirmation that this is a good tip.